concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+14	Bowern-2012-348-14	Magpie	2039	MAGPIE	
+15	Bowern-2012-348-15	Nest	539	NEST	
+16	Bowern-2012-348-16	Possum	470	POSSUM	
+17	Bowern-2012-348-17	Scorpion	1538	SCORPION	
+18	Bowern-2012-348-18	Shield	1064	SHIELD	
+19	Bowern-2012-348-19	Sulphur-Crested Cockatoo	3010	WHITE COCKATOO	
+20	Bowern-2012-348-20	Willy Wagtail	3011	WAGTAIL	
+21	Bowern-2012-348-21	Witchetty Grub	3012	GRUB	
+22	Bowern-2012-348-22	above	1741	ABOVE	
+23	Bowern-2012-348-23	again	204	AGAIN	
+24	Bowern-2012-348-24	all	98	ALL	
+25	Bowern-2012-348-25	alone	1964	ALONE	
+26	Bowern-2012-348-26	and	1577	AND	
+27	Bowern-2012-348-27	angry	1826	ANGRY	check if it shouldn't be BE ANGRY
+28	Bowern-2012-348-28	ankle	803	ANKLE	
+29	Bowern-2012-348-29	ant	587	ANT	
+30	Bowern-2012-348-30	armpit	1886	ARMPIT	
+31	Bowern-2012-348-31	ash	646	ASH	
+32	Bowern-2012-348-32	ask	1309	ASK (INQUIRE)	check if it should be ASK (REQUEST)
+33	Bowern-2012-348-33	at	1461	AT	
+34	Bowern-2012-348-34	back	1291	BACK	
+35	Bowern-2012-348-35	bad/evil	45	EVIL	
+36	Bowern-2012-348-36	bag	2468	BAG	
+37	Bowern-2012-348-37	bandicoot	3013	BANDICOOT	
+38	Bowern-2012-348-38	bat	1793	BAT	
+39	Bowern-2012-348-39	beard	678	BEARD	
+40	Bowern-2012-348-40	belly	1251	BELLY	
+41	Bowern-2012-348-41	below	2094	DOWN OR BELOW	
+42	Bowern-2012-348-42	big	1202	BIG	
+43	Bowern-2012-348-43	bird	937	BIRD	
+44	Bowern-2012-348-44	bite	1403	BITE	
+45	Bowern-2012-348-45	black	163	BLACK	
+46	Bowern-2012-348-46	black cockatoo	3014	BLACK COCKATOO	
+47	Bowern-2012-348-47	black duck	3015	BLACK DUCK	
+48	Bowern-2012-348-48	blind	1653	BLIND	
+49	Bowern-2012-348-49	blood	946	BLOOD	
+50	Bowern-2012-348-50	blow	175	BLOW (OF WIND)	needs to be confirmed
+51	Bowern-2012-348-51	blowfly/housefly	1504	FLY (INSECT)	
+52	Bowern-2012-348-52	body	1480	BODY	
+53	Bowern-2012-348-53	boil/pimple	1809	PIMPLE	
+54	Bowern-2012-348-54	bone	1394	BONE	
+55	Bowern-2012-348-55	boomerang/throwing stick	135	BOOMERANG	
+56	Bowern-2012-348-56	bottom grinding stone	3028	BOTTOM GRINDSTONE	
+57	Bowern-2012-348-57	boy	1366	BOY	
+58	Bowern-2012-348-58	break (intr)	680	BREAK (DESTROY OR GET DESTROYED)	
+59	Bowern-2012-348-59	breast	1402	BREAST	
+60	Bowern-2012-348-60	breathe	1407	BREATHE	
+61	Bowern-2012-348-61	brother-in-law	2314	BROTHER-IN-LAW	
+62	Bowern-2012-348-62	burn	2102	BURN	
+63	Bowern-2012-348-63	butterfly	1791	BUTTERFLY	
+64	Bowern-2012-348-64	calf	1989	CALF	
+65	Bowern-2012-348-65	camp	1105	CAMP	
+66	Bowern-2012-348-66	canoe	1970	CANOE	
+67	Bowern-2012-348-67	cave	1155	CAVE	
+68	Bowern-2012-348-68	charcoal	1188	CHARCOAL	
+69	Bowern-2012-348-69	chase	1435	HUNT	
+70	Bowern-2012-348-70	cheek	1730	CHEEK	
+71	Bowern-2012-348-71	chest	1592	CHEST	
+72	Bowern-2012-348-72	chew	321	CHEW	
+73	Bowern-2012-348-73	child	2099	CHILD	
+74	Bowern-2012-348-74	chin	1510	CHIN	
+75	Bowern-2012-348-75	climb	1102	CLIMB	
+76	Bowern-2012-348-76	cloud	1489	CLOUD	
+77	Bowern-2012-348-77	cold	1287	COLD	
+78	Bowern-2012-348-78	come	1446	COME	
+79	Bowern-2012-348-79	cook	1100	COOK (SOMETHING)	
```

Comment:
Again an insecurity, as it could be the cook, but I think it is rather unlikely for those questionnaires, so the linking is correct, I'd do the same.